skins/laika/tests/unit/components/shared/BankData.spec.ts

```diff
@@ -215,6 +217,83 @@ describe( 'BankData', () => {
 		expect( iban.text() ).toMatch( 'Test Bank' );
 	} );
 
+	it( 'renders bank code / BIC field when legacy bank account number is entered', async () => {
+		const wrapper = mount( BankData, {
+			localVue,
+			store: createStore(),
+			mocks: {
+				$t: () => {},
+			},
+		} );
+
+		let bankInfoInput = wrapper.find( 'input#bic' );
+		expect( bankInfoInput.isVisible() ).toBe( false );
+
+		wrapper.setData( { accountId: '123' } );
+		await wrapper.vm.$nextTick();
+
+		bankInfoInput = wrapper.find( 'input#bic' );
+		expect( bankInfoInput.isVisible() ).toBe( true );
+	} );
+
+	it( 'renders bank code / BIC field when valid IBAN was validated', async () => {
+		let getters;
+		let store;
+		getters = {
+			bankDataIsValid: () => true,
+			bankDataIsInvalid: () => false,
+			getBankName: () => 'gute Bank',
+		}
+		store = new Vuex.Store( {
+			modules: {
+				bankdata: {
+					namespaced: true,
+					getters,
+				},
+			},
+		} );
+		const wrapper = mount( BankData, {
+			localVue,
+			store,
+			mocks: {
+				$t: () => {},
+			},
+		} );
+		wrapper.setData( { accountId: 'DE89370400440532013000' } );
+
+		let bankInfoInput = wrapper.find( 'input#bic' );
+		expect( bankInfoInput.isVisible() ).toBe( true );
+	} );
+
+	it( 'does not render bank code / BIC field if IBAN not validated', async () => {
```

Comment:
Since this test is more about the *state* after the validation and not about the *action* of validating, I'd call this test `does not render bank code / BIC field if IBAN is invalid`. Who do you say as a native-speaker, @Abban ?

Reply:
👍